modules/bringauto_repository/default_test.go

```
@@ -0,0 +1,375 @@
+
+func TestCommitAllChanges(t *testing.T) {
+	repo, err := initGitRepo()
+	if err != nil {
+		t.Fatalf("can't initialize Git repository or struct - %s", err)
+	}
+
+	err = repo.CopyToRepository(pack1, pack1Name)
+	if err != nil {
+		t.Errorf("CopyToRepository failed - %s", err)
+	}
+	repo.CommitAllChanges()
```

golangci-lint (1.62.2)

Error return value of `repo.CommitAllChanges` is not checked

(errcheck)